lib/mayaHydra/hydraExtensions/sceneIndex/mayaHydraSceneIndex.cpp

```diff
@@ -475,6 +477,8 @@ class MayaPathMapper : public Fvp::PathMapper
 
     Fvp::PrimSelections 
     UfePathToPrimSelections(const Ufe::Path& appPath) const override {
+        std::cout << "PPT: in MayaPathMapper::UfePathToPrimSelections("
```

Comment:
This is debugging info that should be removed

Reply:
Good catch!